src/features/home/components/HomeBody.tsx

```diff
@@ -24,60 +26,88 @@ interface HomeBodyProps {
   homeModules: HomeModuleResponse
   recommendedHits: SearchHit[]
   setRecommendationY: (y: number) => void
+  onScroll: (input: any) => void
+}
+
+const keyExtractor = (item: ProcessedModule, index: number) =>
+  // @ts-ignore moduleId is undefined on RecommendationPane
+  item.moduleId || `recommendation${index}`
```

Comment:
```suggestion
  'moduleId' in item ? item.moduleId : `recommendation${index}`
```

et enlève le `@ts-ignore` ?